admin/app/routes/authenticated/organizations/get/team.js

```diff
@@ -13,6 +14,11 @@ export default class OrganizationTeamRoute extends Route {
     organizationRole: { refreshModel: true },
   };
 
+  ERROR_MESSAGES = {
+    DEFAULT: 'Une erreur est survenue.',
+    STATUS_403: "Vous n'avez accès à certaines actions ou informations de cette page",
+  };
```

Comment:
Petit fix rapide pour voir si c'était gérable facilement ou pas.
Récap du problème traité : 
Quand on est sur pix admin, on peut voir la liste des orgas, quand on clique sur l'id d'une orga, on voit la liste des membres de l'orga.
Or, pour afficher la liste des membres d'une orga, on a une seule route api et celle ci filtre sur deux trucs check User Belongs To Organization Or has Role Super Admin
maintenant qu'on a les roles, le membre du support ne passe plus le check du UserBelongsToOrganizationOrhasRoleSuperAdmin (ici user c'est soir un pixou sur admin, soit un orga sur orga) car il n'est pas super admin et il n'appartient pas à l'orga.
Et comme on n'a pas toujours mis en place la gestion des erreurs de l'api ça fait ce qu'on voit à l'écran.
J'ai regardé dans le code, ça se corrige et on peut afficher un message d'erreur.
tu peux aller tester sur https://admin-pr4361.review.pix.fr/ j'ai fait vite fait le fix pour tester si on pouvait corriger ça rapidement. Tu peux te connecter avec  [pixmaster@example.net](mailto:pixmaster@example.net) et pix123 et aller par exemple sur https://admin-pr4361.review.pix.fr/organizations/4/team j'ai fixé avec le message "Vous n'avez accès à certaines actions ou informations de cette page" -> il voit pas les membres mais la page s'affiche correctement.
A priori comme les pixous sont tous super admin ça devrait aller pour le moment.

Reply:
on a corrigé ce problème et mnt le check verifie s'il appartient a l'orga ou s'il est super admin ou certif ou support ou metier

Reply:
Ok, je pensais que on allait mettre à jour les droits rôle par rôle dans les prochains tickets, cette PR était déjà à une centaines de fichiers modifiés